### DISCUSSION TITLE
Include cash balance in overview

### DISCUSSION THREAD

**Discussion by Wibbbs** [ghostfolio/ghostfolio#2308] [Q&A]
Just testing out ghostfolio. I started with just one account with one holding and a cash balance. I noticed that the overview doesn't include the cash balance. Is there anyway to include the cash balance as part of the total overview?

**Comment by dtslvr** [comment]
The overview currently only includes your assets. However, you can see everything in the _Summary_.

**Reply by saugs** [reply]
Maybe just add a second graph on `Overview` page, with `Net worth` history ? As a complement to `Investments`, as for me aim is also to track net worth somewhere in the app.

`Summary` doesn't offer view over the past, allowing to see `Net worth` evolution.

Or maybe add a cumulative area to the `Overview` chart, like that?
<img src="https://github.com/user-attachments/assets/6b583626-ae3f-4366-90c8-e244cf7645d2" />

**Reply by saugs** [reply]
Aim is to see net worth history graph, what do you think? Also cash holdings are part of the overall strategies.

**Comment by Wibbbs** [comment]
I would consider valuables and cash on hand to be an asset. Especially valuables. I would love to make that a feature request as an option. 

Also when I look at summary all I see are ***.

![image](https://github.com/ghostfolio/ghostfolio/assets/24574428/4b02f9b6-6ad0-4a9e-a481-d3d05c08a2ba)

**Reply by dtslvr** [reply]
I also thought about that. I think it's not enough to just add the cash etc. there. The performance in the overview is currently solely based on the assets. How would you integrate the performance of cash there?

It looks like you are using the managed cloud. The _Summary_ is a Ghostfolio Premium feature there.

**Reply by Wibbbs** [reply]
I just installed and setup on my docker server, the summary is working now. Thanks

I don’t see cash as having any performance its just a static number only affected by deposits or withdrawals, I would just add it to the total and not calculate any performance on it at all.

For valuables I would love to be able to manually input/update prices for a valuable over time and have that tracked as performance along with securities.

I own some custom products that are not tracked on any public market but do have a price that is updated daily. I would love to be able to update and track that with periodically manual inputs.

**Reply by dtslvr** [reply]
> I don’t see cash as having any performance its just a static number only affected by deposits or withdrawals, I would just add it to the total and not calculate any performance on it at all.

I do not think that this calculation is sufficient. Here is an example: If you have bought one _Apple_ stock a year ago and hold $100k in cash, it would show a total of $100.178 with a performance of 9.03%. This number would be misleading to me. What do you think?

> For valuables I would love to be able to manually input/update prices for a valuable over time and have that tracked as performance along with securities.
> 
> I own some custom products that are not tracked on any public market but do have a price that is updated daily. I would love to be able to update and track that with periodically manual inputs.

Interesting ideas. Would you like to take on any of these features?

**Reply by prashmohan** [reply]
> I do not think that this calculation is sufficient. Here is an example: If you have bought one Apple stock a year ago and hold $100k in cash, it would show a total of $100.178 with a performance of 9.03%. This number would be misleading to me. What do you think?

I would expect that it considers the cash holding as an explicit asset with no gains or losses. So in your example, I assume the year ago price of the apple stock was 178/1.0903 = $163.25. In this case, I would expect that the dashboard report a performance of 0.14% - (178-178/1.0903)/(100000+178/1.0903)*100

**Reply by GrantBirki** [reply]
> hold $100k in cash, it would show a total of $100.178 with a performance of 9.03%. This number would be misleading to me.

I absolutely agree. Cash should be treated as an asset and displayed like all other holdings

**Reply by GrantBirki** [reply]
@dtslvr Any updates on this? This would be an amazing feature!

**Reply by GrantBirki** [reply]
@dtslvr gentle bump on this 🙇 🙏

**Comment by GrantBirki** [comment]
I absolutely agree that cash and valuables should be considered an asset. I would really love to see them included on the overview page as well!

**Reply by karter16** [reply]
Agree with this. I hold cash as part of an extended emergency fund. I consider this emergency fund to be a part of my overall portfolio and recognise the performance drag that this has on my portfolio as a whole. I would love to be able to include this in Ghostfolio and understand my overall portfolio performance and the impact that that cash has on it.

**Comment by Mirage1337** [comment]
Is this still something that is considered by the dev? 
This is currently the only thing that keeps me at portfolioperformance instead of ghostfolio. 

In the overview and in the performance calculations i would like to see my whole asset allocation and not only my investments. 

if this is not wanted as a default, maybe a toggle in the Admin panel or in the overview tab would be possible?

**Comment by jgancedo** [comment]
Hi all! Since this issue is related to visibility of cash in overview (or in other words, of net wealth), I thought I might post a link to my pull request here:
https://github.com/ghostfolio/ghostfolio/pull/7777

I have added a toggle in the main Overview to select from "Investments" (original view) or Net wealth (new view which includes cash balance history of all added accounts).

Hope it helps! Let me know if you have any questions :)
